test: TestBig_Float is trying to print using wrong type

Any project depending on decimal would get this error when executing `go test all`, specifically:

```
../../../../pkg/mod/github.com/ericlagergren/decimal@v0.0.0-20190121015519-f96f22536576/big_test.go:75:4: Fatalf format %f has arg xf of wrong type *math/big.Float
```

The problem is that `*big.Float` cannot be printed using `%f`as it's a struct.

--- a/big_test.go
+++ b/big_test.go
@@ -72,7 +72,7 @@ func TestBig_Float(t *testing.T) {
 		fv := new(big.Float).SetPrec(flt.Prec())
 		xf := new(decimal.Big).SetFloat(flt).Float(fv)
 		if xf.String() != flt.String() {
-			t.Fatalf("#%d: wanted %s, got %f", i, flt, xf)
+			t.Fatalf("#%d: wanted %s, got %s", i, flt, xf.String())
 		}
 	}
 }

